README.md

```diff
@@ -51,6 +51,15 @@ Running InstaScale locally to an OSD cluster requires extra steps from the above
     - Key: `token`
     - Value: `<YOUR_API_TOKEN>`
     - Click Create
+## Scaling Machines with a Self-Managed OCP Cluster using AWS
+To scale machines of a certain type you need to create a `MachineSet`. You can find a template [here](/template/machineSetTemplate.yaml).
+- Fill out the necessary fields based on your Cluster and AWS configuration.
```

Comment:
Can you link to the existing OpenShift docs here in case people need it?
https://docs.openshift.com/container-platform/4.8/machine_management/creating_machinesets/creating-machineset-aws.html

Reply:
That's actually a lot better than the template idea I'll get on that.

Reply:
One thing to note is that there is no reference to [these](https://github.com/project-codeflare/instascale/pull/142/commits/2702dcdfa174a5e4960d092d70abc8f25b32f85e#diff-51e3d0134e5a8a4d7df4ed8c9a52da04527c7dfd5e34135cc3b0746c0fd230ceR5-R7) annotations in that documentation. I'm not sure if that breaks anything for instascale.

Reply:
The annotations don't seem to matter to InstaScale they were just included in the base MachineSet that I got from my OCP cluster. I don't think it will cause a break if they are left out.